src/Game/NeoServer.Game.Common/Creatures/Players/Gender.cs

```diff
@@ -2,6 +2,6 @@
 
 public enum Gender : byte
 {
-    Male,
-    Female
+    Female,
+    Male
```

Comment:
Can we have a byte assigned to the type?
ie:  Male = 0, Female = 1

Reply:
I think we can change it to: Female = 0, Male = 1 then.
this is probably only used by outfits